backend/index.js

```diff
@@ -15,7 +15,6 @@ async function appStart () {
 		.then(() => {
 			return apiValidator.loadSchemas;
 		})
-		.then(internalIpRanges.fetch)
```

Comment:
Removing this is not good enough to fix the problem. This call might be in the wrong place but you haven't put a better one in the pr.